pkg/cidata/cidata.TEMPLATE.d/network-config

```diff
@@ -6,4 +6,11 @@ ethernets:
       macaddress: '{{$nw.MACAddress}}'
     dhcp4: true
     set-name: {{$nw.Name}}
+    {{- if eq $nw.Name $.SlirpNICName }}
```

Comment:
&& DNSAddress is not empty

Reply:
I'll change it, but I don't think it should make any difference. Setting a key to `nil` or not having the key at all should have the same effect. But having the check means we won't have to worry if there is a bug in dealing with this in cloud-init...

Reply:
[Checking len(DNSAddresses) now](https://github.com/lima-vm/lima/compare/e23912b405ceb80280360fd070db7b4109b7682c..fd4346f725153d7557264038bfccea451550d31f)